recipe/meta.yaml

```diff
@@ -20,19 +20,21 @@ source:
     - osx.patch  # [osx]
 
 build:
-  number: 0
+  number: 1
   detect_binary_files_with_prefix: true
   skip: true  # [win]
 
 requirements:
   build:
     - gcc  # [osx]
+    - libgcc  # [osx]
```

Comment:
`gcc` installs `libgcc`, so this is not really needed (but it does not hurt to leave it there.)